skin.json

```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```

Comment:
What if someone wants to disable dvr entirely from the UI? can we have a field for that?